easybuild/easyconfigs/g/GLib/GLib-2.53.5-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.53.5'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['991421f41a4ed4cc1637e5f9db0d03fd236d2cbd19f3c5b097a343bec5126602']
+
+builddependencies = [
+    ('binutils', '2.28'),
+    ('Python', '2.7.14', '-bare'),
+]
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.8.1'),
+    ('libxml2', '2.9.4'),
+    ('PCRE', '8.41'),
+    ('util-linux', '2.30'),
```

Comment:
@reedts why do you have an earlier version of `util-linux` here compared with the `GCCcore-6.3.0` version?

Reply:
Probably because that easyconfig is already there, while one for `util-linux` 2.30.1 with `GCCcore/6.4.0` isn't there yet.

Makes sense to bump this though, even to the most recent `util-linux` 2.31...